src/main/java/core/basesyntax/dao/impl/CommentDaoImpl.java

```diff
@@ -12,21 +16,63 @@ public CommentDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Comment create(Comment entity) {
```

Comment:
```suggestion
    public Comment create(Comment comment) {
```
and do it everywhere

Reply:
Thanks